roles/openshift_node/tasks/main.yml

```diff
@@ -77,7 +77,7 @@
 - name: Check scheduleable state
   delegate_to: "{{ openshift_first_master }}"
   command: >
-    {{ openshift.common.client_binary }} get node {{ inventory_hostname }}
+    {{ openshift.common.client_binary }} get node {{ openshift_hostname }}
```

Comment:
To answer your question, this can be done with the default filter like this (untested):
```
{{ openshift.common.client_binary }} get node {{ openshift_hostname | default(inventory_hostname, true) }}
```

Or, you could set a fact like this (untested):
```
- name: make sure openshift_hostname is set
  set_fact:
    openshift_hostname: "{{ inventory_hostname }}"
  when: openshift_hostname is not defined or openshift_hostname == ""
```